doc/api/child_process.markdown

```diff
@@ -455,23 +180,31 @@ leaner than [`child_process.exec()`][]. It has the same options.
   * `gid` {Number} Sets the group identity of the process. (See setgid(2).)
 * Return: ChildProcess object
 
-This is a special case of the [`child_process.spawn()`][] functionality for
-spawning Node.js processes. In addition to having all the methods in a normal
-ChildProcess instance, the returned object has a communication channel built-in.
-See [`ChildProcess#send()`][] for details.
+The `child_process.fork()` method is a special case of
+[`child_process.spawn()`][] used specifically to spawn new Node.js processes.
+Like `child_process.spawn()`, a `ChildProcess` object is returned. Unlike
+`child_process.spawn()` however, the returned `ChildProcess` will have an
```

Comment:
"unlike"? whether spawn has a communication channel or not depends on how its stdio option is specified. Better to say that it always has a communication channel, unlike spawn where it may optionally have one.